.pre-commit-config.yaml

```diff
@@ -106,3 +106,13 @@ repos:
         - Babel==2.12.1
         - multipart==0.2.4
         - "git+https://github.com/webpy/webpy.git@d3649322b85777b291ac2b7b3699fb6fc839e382"
+
+  - repo: local
+    hooks:
+    - id: detect-missing-i18n
+      name: Detect missing i18n
+      description: This hook checks .html files for i18n strings that might be missing.
+      entry: python ./scripts/detect_missing_i18n.py
+      types: [html]
+      language: python
+      verbose: true
```

Comment:
I've added this so that `SKIP` lines will still show even when the exit code is 0. Pre-commit will hide the script output if the exit code is 0.